gcc/testsuite/rust/compile/torture/all_doc_comment_line_blocks_crlf.rs

```diff
@@ -9,40 +9,37 @@
 
 /// outer doc line for module
 /** outer doc block for module               */
-pub mod module // { dg-warning "unused name" }
-{
-  //!  inner line doc
-  //!! inner line doc!
-  /*!  inner block doc  */
-  /*!! inner block doc! */
-
-  //   line comment
-  ///  outer line doc
-  //// line comment
-
-  /*   block comment   */
-  /**  outer block doc */
-  /*** block comment   */
-
-  mod block_doc_comments // { dg-warning "unused name" }
-  {
-    /*   /* */  /** */  /*! */  */
-    /*!  /* */  /** */  /*! */  */
-    /**  /* */  /** */  /*! */  */
-    mod item { } // { dg-warning "unused name" }
-  }
-
-  pub mod empty // { dg-warning "unused name" }
-  {
-    //!
-    /*!*/
-    //
-
-    ///
-    mod doc { }    // { dg-warning "unused name" }
-
-    /**/
-    /***/
-  }
+pub mod module {

```

Comment:
minor comment, but looks like you are reindenting automatically here.

Reply:
good find, those edits negated the test case. I have reverted my changes and simply removed the dg-warnings properly.